keyboards/rainkeeb/info.json

```diff
@@ -0,0 +1,52 @@
+{
+    "keyboard_name": "rainkeeb",
+    "url": "",
+    "maintainer": "rain",
+    "width": 12,
+    "height": 4,
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+                {"label":"k00", "x":0, "y":0},
+                {"label":"k01", "x":1, "y":0},
+                {"label":"k02", "x":2, "y":0},
+                {"label":"k03", "x":3, "y":0},
+                {"label":"k04", "x":4, "y":0},
+                {"label":"k05", "x":7, "y":0},
+                {"label":"k06", "x":8, "y":0},
+                {"label":"k07", "x":9, "y":0},
+                {"label":"k08", "x":10, "y":0},
+                {"label":"k09", "x":11, "y":0},
+                {"label":"k10", "x":0, "y":1},
+                {"label":"k11", "x":1, "y":1},
+                {"label":"k12", "x":2, "y":1},
+                {"label":"k13", "x":3, "y":1},
+                {"label":"k14", "x":4, "y":1},
+                {"label":"k15", "x":7, "y":1},
+                {"label":"k16", "x":8, "y":1},
+                {"label":"k17", "x":9, "y":1},
+                {"label":"k18", "x":10, "y":1},
+                {"label":"k19", "x":11, "y":1},
+                {"label":"k20", "x":0, "y":2},
+                {"label":"k21", "x":1, "y":2},
+                {"label":"k22", "x":2, "y":2},
+                {"label":"k23", "x":3, "y":2},
+                {"label":"k24", "x":4, "y":2},
+                {"label":"k25", "x":7, "y":2},
+                {"label":"k26", "x":8, "y":2},
+                {"label":"k27", "x":9, "y":2},
+                {"label":"k28", "x":10, "y":2},
+                {"label":"k29", "x":11, "y":2},
+                {"label":"k30", "x":0, "y":3},
+                {"label":"k31", "x":1, "y":3},
+                {"label":"k32", "x":2, "y":3},
+                {"label":"k33", "x":3, "y":3, "w":2},
+                {"label":"k34", "x":5, "y":3, "w":2},
+                {"label":"k35", "x":7, "y":3, "w":2},
+                {"label":"k36", "x":9, "y":3},
+                {"label":"k37", "x":10, "y":3},
+                {"label":"k38", "x":11, "y":3}
```

Comment:
This will match the updated LAYOUT in rainkeeb.h, but I'm unsure if it's worth reordering the `kXY` to match the macro elements..